layouts/partials/carousel/modal-content.css

```diff
@@ -4,6 +4,8 @@
   {{ end }}
 {{ end }}
 
+"{{ partial (printf "%s/custom-fonts.css" .Type) . }}" +
```

Comment:
Why I can't see a `custom-fonts.css` partial in this PR files?
Need to know why this is needed, thanks.